docker-compose.pdc.prod.yml

```diff
@@ -111,6 +111,23 @@ services:
     depends_on:
       - pdc_strapi
       - pdc_strapi_db
+  strapi-admin-extensions:
+    container_name: strapi-admin-extensions
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    restart: unless-stopped
+    command: yarn strapi-admin-extensions
+    environment:
+      STRAPI_PRIVATE_URL: ${STRAPI_PRIVATE_URL}
+      FRONTEND_PUBLIC_URL: ${FRONTEND_PUBLIC_URL}
+    ports:
+      - "4002:4002"
+    networks:
+      - pdc_strapi_network
+    depends_on:
+      - pdc_strapi
+      - pdc_strapi_db
```

Comment:
Is deze wel direct afhankelijk van `pdc_strapi_db`, of kan deze weg omdat er alleen een indirecte dependency is?

Reply:
Je hebt gelijk, dat kan weg zijn, en  `pdc_strapi` voldoende.